env.example

```diff
@@ -1,3 +1,10 @@
 export LOCATION=eastus
 
+# if not working on the operator
```

Comment:
none of these are referenced in this PR, did you mean to commit this file?

Reply:
Thanks, i'll keep this til later..